dev/benchmarks/macrobenchmarks/test_driver/util.dart

```diff
@@ -40,8 +40,8 @@ void macroPerfTest(
     { Duration pageDelay,
       Duration duration = const Duration(seconds: 3),
       Duration timeout = kTimeout,
-      Future<void> driverOps(FlutterDriver driver),
-      Future<void> setupOps(FlutterDriver driver),
+      Future<void> Function(FlutterDriver driver) driverOps,
```

Comment:
[optional] FYI with this syntax if you prefer to avoid the repeated words you could use `(FlutterDriver)` instead of `(FlutterDriver driver)`. That second "driver" adds no new information. Same with `FlutterProject project`, `FlutterPluginProject pluginProject`. Similarly you could drop the non-information-carrying single letter argument name and use `(Dessert)` over `(Dessert d)`.

The argument names are mandatory with the old syntax, but optional with the function type syntax.

Some of the names are carrying information, like `srcFile` and `destFile`, so those should definitely be retained.